app/views/users/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <p>Users display here</p>
+    <ul>
+    <li class='user'>
+        <div class='photo'>
+            <%= image_tag (@user.photo)%>
+        </div>
+        <div class='user-info'>
+             <h2><%= @user.name %></h2>
+             <p>Number of Posts: <%= @user.posts_counter %></p>
+        </div>
+    </li>
+    <li>
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+    </li>
+    <li>
+        <% @user.recent_posts.each do |post|%>
+        <ul class='posts'>
+<%= link_to user_post_path(@user, post) do %>
+     <div>
+  <p>Post #<%=post.id %></p>
+        </div>
+        <div>
+  <p><%=post.text %></p>
+        </div>
+        <div>
+  <p><%=post.comments_counter %></p>
+  <p><%=post.likes_counter %></p>
+        </div>
+<% end %>
+        </ul>
+        <%end%>
+    </li>
+    </ul>
```

Comment:
- [ ] Kindly consider the show page to match with the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) guidelines
- [ ] Kindly consider adding the `See all posts` link to navigate to user posts

Reply:
Well resolved ✔️

![image](https://github.com/David-Lanzz/lanzz-blog/assets/38088818/122a3cc1-7661-4d03-b528-7348bc282bbe)